<?xml version = "1.0" encoding ="UTF-8"?>
<launch>
  <arg name="model" default="$(find project_opanoid)/urdf/opanoid_sketch.urdf"/>

  <include file="$(find project_opanoid)/launch/gazebo.launch">
    <arg name="model" value="$(arg model)" />
  </include>
  
  <rosparam command="load"
            file="$(find project_opanoid)/config/servos.yaml"
            ns="servo_controller" />
            
	<node name="project_opanoid_node" pkg="project_opanoid" type="project_opanoid_node" args="project_opanoid_node"/>
  <node name="opanoid_controller_spawner" pkg="controller_manager" type="spawner"
    args="servo_controller
          --shutdown-timeout 3"/>
</launch>
